File: codeforces/460A/py/460a.py
import sys

n, m = map(int, sys.stdin.readline().split())

day = 0
mom = 0
while n > 0:
    day += n
    mom += n
    if mom >= m:
        n = int(mom / m)
        mom -= n*m
    else:
        break

print(day)

File: codeforces/460A/go/460a.go
package main

import "fmt"

func main() {
    var m, n int
    fmt.Scan(&n, &m)

    day := 0
    mom := 0
    for n > 0 {
        day += n
        mom += n
        if mom >= m {
            n = mom / m
            mom -= m*n
        } else {
            break
        }
    }

    fmt.Println(day)
}
